src/views/SearchPage.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div>
+    <v-header>
+      <v-search />
+    </v-header>
+    <v-results />
+    <v-movie-grid></v-movie-grid>
+  </div>
+</template>
+
+<script>
+import vResults from "../components/v-results";
+import vHeader from "../components/v-header";
+import vSearch from "../components/v-search";
+import vMovieGrid from "../components/v-movie-grid";
+
+export default {
+  name: "MainView",
```

Comment:
Naming a component and its SFC differently might lead to confusion in code base and in dev tools.